auto_submit/test/requests/update_revert_reviews_test.dart

```diff
@@ -0,0 +1,176 @@
+        githubService: fakeGithubService,
+      );
+
+      expect(result, isFalse);
+    });
+
+    test('Closed issue is updated.', () async {
+      User user = User(login: 'ricardoamador');
+      Issue issue = Issue(
+        user: user,
+        state: 'closed',
+        closedAt: DateTime.now(),
+        number: 1234,
+        closedBy: user,
+      );
+
+      fakeGithubService.githubIssueMock = issue;
+
+      RevertRequestRecord revertRequestRecord = RevertRequestRecord(
+        reviewIssueAssignee: 'keyonghan',
+        reviewIssueCreatedTimestamp: DateTime.now(),
+        reviewIssueNumber: 1234,
+      );
+
+      when(jobsResource.query(captureAny, expectedProjectId)).thenAnswer((Invocation invocation) {
+        return Future<QueryResponse>.value(
+          QueryResponse.fromJson(jsonDecode(updateSuccessResponse) as Map<dynamic, dynamic>),
+        );
+      });
+
+      bool result = await updateRevertReviews.updateIssue(
+        revertRequestRecord: revertRequestRecord,
+        bigqueryService: fakeBigqueryService,
+        githubService: fakeGithubService,
+      );
+
+      expect(result, isTrue);
+    });
+
+    test('Closed issue is not updated on exception.', () async {
+      User user = User(login: 'ricardoamador');
+      Issue issue = Issue(
+        user: user,
+        state: 'closed',
+        closedAt: DateTime.now(),
+        number: 1234,
+        closedBy: user,
+      );
+
+      fakeGithubService.githubIssueMock = issue;
+
+      RevertRequestRecord revertRequestRecord = RevertRequestRecord(
+        reviewIssueAssignee: 'keyonghan',
+        reviewIssueCreatedTimestamp: DateTime.now(),
+        reviewIssueNumber: 1234,
+      );
+
+      when(jobsResource.query(captureAny, expectedProjectId)).thenAnswer((Invocation invocation) {
+        throw BigQueryException('Update of review issue 1234 did not complete.');
+      });
+
+      bool result = await updateRevertReviews.updateIssue(
+        revertRequestRecord: revertRequestRecord,
+        bigqueryService: fakeBigqueryService,
+        githubService: fakeGithubService,
+      );
+
+      expect(result, isFalse);
+    });
+  });
+
+  group('Update closed revert review requests.', () {
+    test('All review requests are processed successfully', () async {
+      when(jobsResource.query(captureAny, expectedProjectId)).thenAnswer((Invocation invocation) {
+        return Future<QueryResponse>.value(
+          QueryResponse.fromJson(jsonDecode(emptyRowsResponse) as Map<dynamic, dynamic>),
+        );
+      });
+
+      Response response = await updateRevertReviews.get();
+      expect(response.statusCode, 200);
+      String body = await response.readAsString(Encoding.getByName("UTF-8"));
+      expect(body, equals('No open revert reviews to update.'));
+    });
+
+    test('Updateable review requests are process successfully.', () async {});
+
+    test('No review requests to process is successful.', () async {});
+  });
```

Comment:
Do these need to get implemented, or shall we remove?

Reply:
Removed. These cannot be verified easily.